rasa_nlu/classifiers/regex_intent_classifier.py

```diff
@@ -0,0 +1,58 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+
+from builtins import map
+from typing import Any
+from typing import Dict
+from typing import Text
+import re
+
+import rasa_nlu.converters as converters
+from rasa_nlu.training_data import TrainingData
+
+from rasa_nlu.components import Component
+
+
+class RegExIntentClassifier(Component):
+
+    name = "intent_classifier_regex"
+
+    context_provides = {
+        "process": ["intent"],
+    }
+
+    output_provides = ["intent"]
+
+    def __init__(self, clf=None, regex_dict=None):
+        self.clf = clf
+        self.regex_dict = {}
+
+    def train(self, training_data):
+    # build regex: intent dict from training data
```

Comment:
Should be a function comment, e.g. `"""Build regex: ..."""` instead of `# ...` describing the functionality of the function.